nova/console/shellinaboxproxy.py

```diff
@@ -40,9 +52,10 @@ def response(self, flow):
             else:
                 # Validate the token
                 ctxt = context.get_admin_context()
-                rpcapi = consoleauth_rpcapi.ConsoleAuthAPI()
-
-                if not rpcapi.check_token(ctxt, token=self.token):
+                try:
+                    super(NovaShellInaBoxProxy, self)._get_connect_info(
+                        ctxt, self.token)
```

Comment:
For consoleauth enabled, this just returns a boolean like before and doesn't raise an exception: https://github.com/sapcc/nova/blob/c49cb948bd38ef558c763ddde6db517068579d19/nova/console/websocketproxy.py#L129-L134

Reply:
Yes, but if it's false it falls under the DB call, which finally throws an exception.

https://github.com/sapcc/nova/blob/c49cb948bd38ef558c763ddde6db517068579d19/nova/console/websocketproxy.py#L172-L173

Reply:
Ah, you're right.